src/DynamoSandbox/Program.cs

```diff
@@ -102,5 +112,30 @@ private static void UpdateSystemPathForProcess()
                         EnvironmentVariableTarget.Process) + ";" + DynamoCorePath;
             Environment.SetEnvironmentVariable("Path", path, EnvironmentVariableTarget.Process);
         }
+        
+        /// <summary>
+        /// If Dynamo Sandbox fails to acquire Dynamo Core path, show a dialog that
+        /// redirects to download DynamoCore.msi, and the program should exit.
+        /// </summary>
+        private static void NotifyUserDynamoCoreUnresolved()
```

Comment:
Can we move this method to DynamoInstallDetective? The string needs to be localized.